contracts/extension/NFTMetadata.sol

```diff
@@ -4,8 +4,10 @@ pragma solidity ^0.8.0;
 import "./interface/INFTMetadata.sol";
 
 abstract contract NFTMetadata is INFTMetadata {
+
+    bool internal _URIFrozen;
```

Comment:
Since this variable is internal, add a view function that can be called to check whether metadata is frozen or not

Reply:
Marked as public and renamed to URIFrozen.